apps/cli/cli_common.c

```diff
@@ -774,6 +775,125 @@ cli_set_mode(clixon_handle h,
     return retval;
 }
 
+/*!
+ * @brief Spawns a new process and runs the specified program within it
+ *
+ * The function creates a new child process and runs the specified program within it.
+ * Before launching the program in the child process, environment variables taken from cvv are set.
+ *
+ * The function checks the parameters passed to it. Situations where at least one function argument
+ * is missing, or when two arguments are present in both the function and the cvv vector are considered invalid.
+ *
```

Comment:
Please add a warning about safety here, essentially you could start any script from the command-line if you use some variants here. That may be OK, but a designer may also want to chroot after the fork to limit the space of accessible scripts.

Reply:
done